src/data/preprocessing.py

```diff
@@ -0,0 +1,62 @@
+from ast import literal_eval
+import re
+import os
+import pandas as pd
+import numpy as np
+
+import nltk
+from nltk.corpus import stopwords
+
+nltk.download("stopwords")
```

Comment:
Maybe move this to the top of the `main()` function?

Reply:
I don't know if that fixes it. Can't there be cases where you need to import smth like the `text_prepare` function without calling `main()`? For example you already have all the data from DVC, but you need to call `text_prepare()` for the inference endpoint.

Another idea would be to have a singleton object here for the stopwords, but not really. I can try this

Reply:
Yeah true, you are right